src/core/execute.c

```diff
@@ -232,7 +232,7 @@ bool exec_needs_ipc_namespace(const ExecContext *context) {
         return context->private_ipc || context->ipc_namespace_path;
 }
 
-static bool can_apply_cgroup_namespace(const ExecContext *context, const ExecParameters *params) {
+static bool can_apply_cgroup_namespace(const ExecContext *context) {
         return cg_all_unified() > 0 && ns_type_supported(NAMESPACE_CGROUP);
```

Comment:
Hmm, if you're touching this, just inline the ns_type_supported call in exec_get_protect_control_groups? cgv1 has been fully disabled after all